client/blocks/reader-subscription-list-item/style.scss

```diff
@@ -208,3 +215,40 @@
 	color: lighten( $gray, 10% );
 	display: block;
 }
+
+// Placeholders
+.reader-subscription-list-item.reader-subscription-list-item__placeholder {
+	.reader-subscription-list-item__site-title,
+	.reader-subscription-list-item__site-excerpt,
+	.reader-subscription-list-item__by-text,
+	.reader-subscription-list-item__site-url,
+	.reader-subscription-list-item__follow {
+		@include placeholder();
+	}
+
+	.reader-subscription-list-item__site-excerpt {
+		margin-top: 2px;
+	}
+
+	.reader-subscription-list-item__site-title {
+		max-width: 60%;
+	}
+
+	.reader-subscription-list-item__site-excerpt {
+		max-width: 80%;
+	}
+
+	.reader-subscription-list-item__site-url {
+		max-width: 40%;
+		margin-top: -1px;
+	}
+
+	.reader-subscription-list-item__follow {
+		width: 20px;
+		height: 20px;
+
+		@include breakpoint( ">660px" ) {
+			width: 72px;
```

Comment:
Sound like we're best of with `16px * 1.5 * 3` then for consistency's sake. Changed :)